imports/plugins/core/accounts/client/components/addressBook.js

```diff
@@ -0,0 +1,397 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { Components, registerComponent } from "@reactioncommerce/reaction-components";
+
+class AddressBook extends Component {
+  static propTypes = {
+    /**
+     * Add address reducer calls meteor method
+     */
+    addAddress: PropTypes.func,
+    /**
+     * array of address objects
+     */
+    addressBook: PropTypes.arrayOf(PropTypes.shape({
+      _id: PropTypes.String,
+      fullName: PropTypes.String,
+      address1: PropTypes.String,
+      addresss2: PropTypes.String,
+      postal: PropTypes.String,
+      city: PropTypes.String,
+      region: PropTypes.String,
+      country: PropTypes.String,
+      phone: PropTypes.String,
+      isBillingDefault: PropTypes.Bool,
+      isShippingDefault: PropTypes.Bool,
+      isCommercal: PropTypes.Bool
+    })),
+    /**
+     * country options for select
+     */
+    countries: PropTypes.arrayOf(PropTypes.shape({
+      label: PropTypes.String,
+      value: PropTypes.String
+    })),
+    /**
+     *  Heading content for address book
+     */
+    heading: PropTypes.shape({
+      /**
+       * Heading title
+       */
+      defaultValue: PropTypes.String,
+      /**
+       * i18nKey for heading title
+       */
+      i18nKey: PropTypes.String,
+      /**
+       * If in checkout view, addressbook checkout step position and icon className
+       */
+      checkout: PropTypes.shape({
+        icon: PropTypes.String,
+        position: PropTypes.Number
+      })
+    }),
+    /**
+     * handles error by calling Alerts.toast with the error meesage
+     */
+    onError: PropTypes.func,
+    /**
+     * regions by county
+     */
+    regionsByCountry: PropTypes.shape({
+      countryCode: PropTypes.arrayOf(PropTypes.shape({
+        label: PropTypes.String,
+        value: PropTypes.String
+      }))
+    }),
```

Comment:
This prop type is wrong. There isn't a key named "countryCode" but rather one arbitrary key for each country code. So since this is a map, it's OK to do `regionsByCountry: PropTypes.object` instead of shape.